tools/readme_generator/make_readme.py

```diff
@@ -51,6 +51,16 @@ def generate_READMEs(app_path: Path):
 
     env = Environment(loader=FileSystemLoader(Path(__file__).parent / "templates"))
 
+    screenshots: List[str]
+    screenshots = []
```

Comment:
Why do this on two separate lines ?

Reply:
I did this by default as I didn't know if python 3.9 supported it